openedx_events/event_bus/avro/tests/schemas/org+openedx+learning+program+certificate+revoked+v1_schema.avsc

```diff
@@ -0,0 +1,96 @@
+{
+  "name": "CloudEvent",
+  "type": "record",
+  "doc": "Avro Event Format for CloudEvents created with openedx_events/schema",
+  "fields": [
+    {
+      "name": "program_certificate",
+      "type": {
+        "name": "ProgramCertificateData",
+        "type": "record",
+        "fields": [
+          {
+            "name": "user",
+            "type": {
+              "name": "UserData",
+              "type": "record",
+              "fields": [
+                {
+                  "name": "id",
+                  "type": "long"
+                },
+                {
+                  "name": "is_active",
+                  "type": "boolean"
+                },
+                {
+                  "name": "pii",
```

Comment:
whoa neat is `pii` a data type in this schema?

Reply:
The `UserData` data class is made up of non-PII and PII data. Relevant links: 

UserData: https://github.com/openedx/openedx-events/blob/c0eb4ba1a3d7d066d58e5c87920b8ccb0645f769/openedx_events/learning/data.py#L45
UserNonPersonalData: https://github.com/openedx/openedx-events/blob/c0eb4ba1a3d7d066d58e5c87920b8ccb0645f769/openedx_events/learning/data.py#L15
UserPersonalData: https://github.com/openedx/openedx-events/blob/c0eb4ba1a3d7d066d58e5c87920b8ccb0645f769/openedx_events/learning/data.py#L29